### DISCUSSION TITLE
Open telemetry not working with NextJS (https://opentelemetry.io/docs/instrumentation/js/getting-started/browser/)

### DISCUSSION THREAD

**Discussion by chethan-devopsnow** [vercel/next.js#32206] [Help]
### What version of Next.js are you using?

latest

### What version of Node.js are you using?

16

### What browser are you using?

google chrome

### What operating system are you using?

mac

### How are you deploying your application?

local

### Describe the Bug

Hello Team,

I am trying to get Open telemetry working with Nextjs but not able to get it working. Same option works really well with a standalone project but fails for a nextjs / react project. Has anyone tried setting open telemetry with nextjs if yes could you please guide towards the setup process ?

https://opentelemetry.io/docs/instrumentation/js/getting-started/browser/

### Expected Behavior

Nextjs should work with Open Telemetry to facilitate browser auto instrumentation.

### To Reproduce

https://opentelemetry.io/docs/instrumentation/js/getting-started/browser/

**Comment by chethan-devopsnow** [comment]
I have logged a bug in the Open Telemetry repo as well (https://github.com/open-telemetry/opentelemetry-js-contrib/issues/717). Since there were no response trying to check here if anyone would have tried implementing the same.

**Comment by efraa** [comment]
??
